decidim-core/lib/decidim/assets/tailwind/tailwind.config.js.erb

```diff
@@ -47,21 +47,32 @@ module.exports = {
         3: "#E1E5EF",
         4: "#242424",
         5: "#F6F8FA",
-        6: "#D3D5D9"
+        6: "#D3D5D9",
+        7: "#D4D4D4",
+        8: "#737373",
+        9: "#F5F5F5"
```

Comment:
This should be removed and its usages replaced with `neutral-100`

Reply:
Added a reminder here: https://github.com/decidim/decidim/issues/15299#issuecomment-3380196515